src/app/user/user.component.less

```diff
@@ -0,0 +1,13 @@
+.mat-card {
+  width: 70%;
+  margin: 0 auto;
+}
+
+.example-card {
+  max-width: 400px;
+}
+
+.example-header-image {
+  background-image: url("https://material.angular.io/assets/img/examples/shiba1.jpg");
+  background-size: cover;
```

Comment:
better to use an image HTML tag, do not set it by background-image if it possible to do by HTML